<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeAnimation</class>
 <widget class="QMainWindow" name="AttributeAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attribute Animation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>291</width>
      <height>314</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_attributeAnimation">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Attribute Animation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="list_objects"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="VL_buttons">
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HL_attributeName">
       <item>
        <widget class="QLabel" name="lbl_attributeName">
         <property name="text">
          <string>Attribute Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_attributeName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HL_attributeName_3">
       <item>
        <widget class="QLabel" name="lbl_animStartTime">
         <property name="text">
          <string>Anim Start Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_startTime"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_animFinishTime">
         <property name="text">
          <string>Anim Finish Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_finishTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HL_attributeName_2">
       <item>
        <widget class="QLabel" name="lbl_attributeName_2">
         <property name="text">
          <string>Control Object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_controlObject"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_useSelected">
         <property name="text">
          <string>Use Selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create">
       <property name="text">
        <string>Create Attribute Animation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
